projects/plugins/jetpack/modules/ai.php

```diff
@@ -0,0 +1,23 @@
+<?php
+/**
+ * Module Name: Jetpack AI
+ * Module Description: Add Jetpack AI to your site and post editors
+ * Sort Order: 44
+ * First Introduced: 14.2.0-a.1
```

Comment:
This can be 14.2 I think.

```suggestion
 * First Introduced: 14.2
```